src/bits64/vmx.rs

```diff
@@ -0,0 +1,94 @@
+//! Virtualize processor hardware for multiple software environments using Virtual Machine Extensions.
+
+use crate::bits64::rflags::{self, RFlags};
+use crate::vmx::{Result, VmFail};
+
+/// Helper used to extract VMX-specific Result in accordance with
+/// conventions described in Intel SDM, Volume 3C, Section 30.2.
+#[inline(always)]
+fn vmx_capture_status() -> Result<()> {
+    let flags = unsafe { rflags::read() };
```

Comment:
>Is my guess right that this has to be inline because of the rflags::read(). If so a comment would be good for it.

Exactly, added in 19f141a5aa1bdfa3eef0d381f27ade156748e824.